Makefile

```diff
@@ -71,6 +71,10 @@ run-hc:
 		-e OVERLAY_PORT=7070 \
 		hc
 
+test-captplanet:
```

Comment:
Well actually, it will currently exit with something like `-bash: captplanet: command not found` if the binary does not exist, which will cause the test to fail. Do you think we should still verify that we have the binary?

Reply:
I meant we do the install of the binary here so we always have it when we run this

Reply:
ok, I can do that. But I'm still gonna install the aws cli in the travis deps section since there's some weird stuff I had to do to get that working

Reply:
I added the captplanet installation to the top of the `test-captplanet.sh` script